### DISCUSSION TITLE
Wildcard domain support for next/image

### DISCUSSION THREAD

**Discussion by okose-me** [vercel/next.js#23909] [Ideas]
### Describe the feature you'd like to request

Hello, our project works on more than one domain, we need to add more than one address as a subdomain when we use next / image.

E.g .:
us.example.com
de.example.com
es.example.com

as.

### Describe the solution you'd like

Can wildcard support be added instead?

Eg:
*.example.com

### Describe alternatives you've considered

Couldn't find an alternative solution :)

**Comment by Saulius-HALO** [comment]
I am wandering about this too.

**Reply by FDiskas** [reply]
Example PR created https://github.com/vercel/next.js/pull/27345
